### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by kotielnikovIvan**
Integration tests created for login and register user.

### CODE REVIEW

File: integration/pom.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <artifactId>web</artifactId>
+            <groupId>fit-goal</groupId>
+        </dependency>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-testing</artifactId>
```
Comment (Ivashchyk) [inline integration/pom.xml:25]: Need to set `test` scope for each dependency

File: integration/src/test/java/com/fitgoal/integration/AbstractIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    @AfterClass
+    public static void tearDown() {
+        appExtension.after();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        SqlSessionManager sessionManager = SqlSessionManager.newInstance(getSqlSessionFactory());
+        dao = new UserDaoImpl(sessionManager);
```
Comment (Ivashchyk) [inline integration/src/test/java/com/fitgoal/integration/AbstractIntegrationTest.java]: Can't this be done in a static context as well?

Comment (Ivashchyk) [reply integration/src/test/java/com/fitgoal/integration/AbstractIntegrationTest.java]: In general, once you have test helper class, so I would move this functionality there

**Review by Ivashchyk — CHANGES_REQUESTED** [review]
Minor issues

File: integration/pom.xml (hunk 1 above)
Comment (kotielnikovIvan) [reply integration/pom.xml:25]: Fixed

File: integration/src/test/java/com/fitgoal/integration/AbstractIntegrationTest.java (hunk 2 above)
Comment (kotielnikovIvan) [reply integration/src/test/java/com/fitgoal/integration/AbstractIntegrationTest.java]: Refactored

**Review by shalagatskyi — APPROVED** [review]
good job!

**Pull request merged**